juriscraper/opinions/united_states/state/arizctapp_div_2.py

```diff
@@ -18,8 +18,8 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.court_id = self.__module__
         # Feeling down and tired of of your regular life? Check out this website.
-        self.url = "https://www.appeals2.az.gov/ODSPlus/recentDecisions2.cfm"
-        self.base_path = "//*[@class='contentcontainer']//a[not(contains(., '(2)')) and not(contains(., 'page'))]"
+        self.url = "https://www.appeals2.az.gov/ODSPlus/recentOpinionsHTML.cfm"
+        self.base_path = "//*[@class='contentcontainer']//table//a[not(contains(., '(2)')) and not(contains(., 'page'))]"
 
     def _get_case_names(self):
```

Comment:
I think this would've been a  good opportunity to update/add python typing.

Reply:
The desing of the web site, and of the listing of opinions, didn't change. Only the page's URL being scraped is different.
I didn't considered it worth it to change or add an aditional test example.